cmd/fluxctl/identity_cmd.go

```diff
@@ -22,6 +24,8 @@ func (opts *identityOpts) Command() *cobra.Command {
 		RunE:  opts.RunE,
 	}
 	cmd.Flags().BoolVarP(&opts.regenerate, "regenerate", "r", false, `Generate a new identity`)
+	cmd.Flags().BoolVarP(&opts.fingerprint, "fingerprint", "l", false, `Show fingerprint of public key`)
```

Comment:
That's what `ssh-keygen` uses for the same purpose. I don't know why; but, `-f` would be bad because it usually means "file".